docs/src/modules/components/ApiPage.js

```diff
@@ -221,14 +222,15 @@ export default function ApiPage(props) {
     >
       <MarkdownElement>
         <h1>{pageContent.name} API</h1>
-        <Typography
-          variant="h5"
-          component="p"
-          className={`description${disableAd ? '' : ' ad'}`}
-          gutterBottom
-        >
+        <Typography variant="h5" component="p" className="description" gutterBottom>
           {description}
-          {disableAd ? null : <Ad />}
+          {disableAd ? null : (
+            <BrandingProvider>
+              <AdGuest>
+                <Ad />
+              </AdGuest>
+            </BrandingProvider>
+          )}
```

Comment:
Since the two other `<Ad />` are used with the same wrapper. Would it make sens to rename `Ad` by `AddContent`

And define `<Ad />` as follow:

```jsx
const Ad = ({ disableAd }) => {
  if (disableAd) {
    return null;
  }
  
  return  <BrandingProvider>
              <AdGuest>
                <AddContent />
              </AdGuest>
            </BrandingProvider>
}

Reply:
We could bundle `<AdGuest>` and `<Ad>` together.

I wouldn't bundle `<BrandingProvider>` because this one shouldn't be here but set higher in the tree.